CLI/Testing/Cxx/RegressionTests.cmake

```diff
@@ -2,8 +2,8 @@
 # Convenience functions to get command snippets needed for most tests
 #-----------------------------------------------------------------------------
 function(set_compareArgs computeFpv)
-  set(compareArgs --compareIntensityTolerance 1e-5
-                  --compare ${referenceDataBaseName}-conc.nrrd
+  set(compareArgs --compareIntensityTolerance 1e-4
+                  --compare ${referenceDataBaseDir}${testName}-conc.nrrd
```

Comment:
I think this should be "--compare ${referenceDataBaseName}-conc.nrrd"
It's one line from master and one line from this branch to merge correctly

Reply:
Ah, ok - thanks! Fixing this from command line now.